dart/lib/src/sentry_tracer.dart

```diff
@@ -63,11 +65,13 @@ class SentryTracer extends ISentrySpan {
       // finish unfinished spans otherwise transaction gets dropped
       for (final span in _children) {
         if (!span.finished) {
-          await span.finish(status: SpanStatus.deadlineExceeded());
+          await span.finish(
+              status: SpanStatus.deadlineExceeded(),
+              endTimestamp: endTimestamp);
```

Comment:
m: I believe we should have an auxiliary variable at the beginning of this function to hold the end value ( `endTimestamp ?? getUtcDateTime()` ).
This way, every unfinished span will have the same endTimestamp as well the `_rootEndTimestamp`